DuggaSys/microservices/duggaedService/deleteDuggaVariant_ms.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+date_default_timezone_set("Europe/Stockholm");
+
+// Include basic application services!
+include_once "../../../Shared/sessions.php";
+include_once "../../../Shared/basic.php";
+
+pdoConnect();
+session_start();
+
+$opt=getOP('opt');
+$vid=getOP('vid');
+
+if(strcmp($opt,"DELVARI")===0){
+    $query = $pdo->prepare("DELETE FROM userAnswer WHERE variant=:vid;");
+    $query->bindParam(':vid', $vid);
+
+    if(!$query->execute()) {
+        $error=$query->errorInfo();
+        $debug="Error updating user".$error[2];
```

Comment:
this debug output would maybe be more clear:
```suggestion
        $debug="Error when trying to delete user answer".$error[2];
```